lib/process-services/app-list/apps-list.component.ts

```diff
@@ -50,10 +52,29 @@ export class AppsListComponent implements OnInit, AfterContentInit {
     @Input()
     filtersAppId: any[];
 
+    /* Toggles multiple row selection, renders checkboxes at the beginning of each row */
+    @Input()
+    multiselect: boolean = false;
```

Comment:
The problem that I see with this solution is that we try to merge two things togheter that are actually really different. So when you seee the apps as tails most of those property are not possible to use. It will be difficult explain it.
I guess we need to split this different view it in two different component. Or make available the same options even for the tiles view